css/style.css

```
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20px;
+  line-height: 1.5;
+}
+
+header {
+  font-size: 20px;
+  margin: 0 auto;
+  text-align: center;
+  background-color: lightgreen;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+#tribute-info {
+  background-color: lightgreen;
```

```suggestion
  background-color: #90ee90;
```
Please refer to the same optional advice in the earlier comment on color use in CSS with the articles for reference.

---

It is great that you avoided redundantly explaining changes :+1: .  Perhaps adding the line number of the previous comment would further help the student in finding the required change. :recycle: